enqueue/fs/0.3/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/",
+        "var/": "%VAR_DIR%"
+    },
+    "env": {
+        "ENQUEUE_FS_STORAGE": "%VAR%/queue"
+    },
+    "gitignore": [
+        "/var/queue"
```

Comment:
Framework bundle is a requirement, it cannot be uninstalled or not installed, so this is safe to remove.